### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by redrummed1** [Courseplay/courseplay#1540]
This might sound crazy with all your work in getting the tractor to stay off crops, but anyway here goes: 

Could there be an option put in to have the tractor turn within the field boundaries when on combi mode, or even harvesters, tractors doing fieldwork.  My main problem is I play on small to medium fields with hedges so I have to babysit tractors on combi mode as nearly 99% they get stuck in a tree or ditch trying to turn. 

Obviously crop destruction would be turned off! Is an option like this possible where it recognizes the field edge and makes its turn within it??

**Comment by SquareCloud506** [comment]
With FS15 and CP, I would use up to 4 headlands on maps like Coldbourogh Park (probably spelled that wrong). With a chaff harvester I'd use 6 headlands and manually turn down the turn radius of following tractors to attempt to keep them on the field.

**Comment by redrummed1** [comment]
Headlands help great for a grain harvester but chaff is a different story ,can get very frustrating! what you have suggested is a good idea but essentially you are babysitting them at the end of the day, I would rather be at the pit leveling and rolling the pit then reversing them out of trees.

**Comment by SquareCloud506** [comment]
Yeah, even with 6 headlands a tractor following a chaff harvester would still take 40 acres to turn at the end of a row, and still drives out of the field a bit. I could see what they were doing, swinging wide so they could drive to a specific location behind the harvester before choosing to follow right or left, and in order to hit that location, they would exit the field so they could drive straight at it, instead of coming at an angle. That's why I tried manually lowering the turn radius, but turn it too low, and then they'd stop "in traffic" behind the combine, lol. 

Chaff harvesting does seem to work best on maps with large fields and wide open spaces around the edges. :)

**Comment by redrummed1** [comment]
Tell me about it!!  followme mod does a good job but you have to drive the harvester yourself, I play on British maps so there's always going to be obstacles.  Hopefully the guys on here can come up with something

**Comment by pops64** [comment]
#1469
